templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,16 @@
+---
+services:
+  sonda:
+    container_name: '{{ nim_waku_sonda_cont_name }}'
+    build:
+      context: .
+      dockerfile: Dockerfile.sonda
+    ports:
+      - {{ nim_waku_sonda_metrics_port }}:{{ nim_waku_sonda_metrics_port }}
+    env_file: "sonda.env"
+    networks:
+      - {{ nim_waku_cont_name }}_default
+
+networks:
+  {{ nim_waku_cont_name }}_default:
+    external: true
```

Comment:
Unless we are intending to combine two or more containers with this I don't think we need this setting.

Reply:
> Unless we are intending to combine two or more containers with this I don't think we need this setting.

I need this since we are calling the waku node by container name so I need sonda container to be on the same docker network

Reply:
Why are we calling it by container name tho?

Reply:
Because this port which we are calling is not open to public, so I didn't know any better way than doing it like this

Reply:
We could just open it within WireGuard VPN. Combining containers under one network is a bad idea unless they are really tightly coupled. Seems to me like this is overcomplicating it.